python/dgl/data/sbm.py

```diff
@@ -62,40 +72,110 @@ class SBMMixture:
     n_communities : int
         Number of communities.
     k : int, optional
-        Multiplier.
+        Multiplier. Default: 2
     avg_deg : int, optional
-        Average degree.
+        Average degree. Default: 3
     pq : list of pair of nonnegative float or str, optional
-        Random densities.
+        Random densities. Default: Appendix_C
```

Comment:
I'm confused by this `Appendix_C`. What are the options for `str` input, and what does a float input mean?

Reply:
I'm also confused by this, maybe it's for future extension, it's not functional by now.

Reply:
I think we can remove it.

Reply:
Removing it may break already existing examples/users' own code. Probably we add a note here.